SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/GenerateSchemaAndSnapshotsCommandlet.h

```diff
@@ -20,8 +20,15 @@ class UGenerateSchemaAndSnapshotsCommandlet : public UCommandlet
 	virtual int32 Main(const FString& Params) override;
 
 private:
-	void GenerateSchema(FSpatialGDKEditor& SpatialGDKEditor);
-	void GenerateSnapshots(FSpatialGDKEditor& SpatialGDKEditor);
-	TArray<FString> GetAllMapPaths(FString InMapsPath);
-	void GenerateSnapshotForMap(FSpatialGDKEditor& SpatialGDKEditor, FString WorldPath);
+	const FString MapPathsParamName = TEXT("MapPaths");	// Commandline Argument Name used to declare the paths to generate schema/snapshots against
+	const FString GameDirName = TEXT("/Game");			// Root directory name that maps will ultimately be found in
```

Comment:
The name of the variable can be misleading since it's the asset path (or package path) as opposed to the actual directory path in the file system.

Reply:
Which variable?
MapPathsParamName: Are you saying we should change "MapPaths" to something more descriptive to the user?
GameDirName: How so? The variable doesn't even imply it's a path, only a directory name -- which is the same regardless of whether it's used as a part of an asset/package path, or a file system path.

Reply:
I was referring to the `GameDirName`, which seems ambiguous exactly because it could be either an asset/package path or a file system path, but in fact is only ever an asset/package path.